test/Microsoft.Azure.SignalR.Tests/NegotiateHandlerFacts.cs

```diff
@@ -357,6 +358,8 @@ public override ServiceEndpoint GetNegotiateEndpoint(HttpContext context, IEnume
                 {
                     context.Response.StatusCode = 400;
                     var response = Encoding.UTF8.GetBytes("Invalid request");
+                    // In latest DefaultHttpContext, response body is set to null
+                    context.Response.Body = new MemoryStream();

```

Comment:
check if it is null first?

Reply:
Directly access Body to check null will return ObjectNullReference exception.

Reply:
Looks in AspNetCore tests they just new MemoryStream() for the DefaultHttpContext(). Let's follow it.
https://github.com/aspnet/AspNetCore/blob/master/src/SignalR/common/Http.Connections/test/ServerSentEventsTests.cs#L119